src/uct/api/uct.h

```diff
@@ -1781,6 +1876,17 @@ void uct_iface_close(uct_iface_h iface);
 ucs_status_t uct_iface_query(uct_iface_h iface, uct_iface_attr_t *iface_attr);
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Get interface performance attributes, by memory types and operation
```

Comment:
add details: that user should fill op, memtype, etc and then will have overhead&bandwidth as the output